### ISSUE TITLE
Bug: Differently sized pages

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fleszar1/flexpaper#130]
```
The player seems to assume all pages are the same size upon initial loading, 
even if they are not.  

Temporary fix:

function onDocumentLoaded() {
    getDocViewer().switchMode("Tile");
    getDocViewer().switchMode("Portrait");
}
```

Original issue reported on code.google.com by `support%...@gtempaccount.com` on 18 Oct 2010 at 3:20

**Comment by GoogleCodeExporter** [comment]
```
Example file
```

Original comment by `support%...@gtempaccount.com` on 18 Oct 2010 at 3:24

Attachments:
- [preview.swf](https://storage.googleapis.com/google-code-attachments/flexpaper/issue-130/comment-1/preview.swf)

**Comment by GoogleCodeExporter** [comment]
```
which version of FlexPaper are you running, I recall this being fixed sometime 
earlier
```

Original comment by `erik.eng...@devaldi.com` on 20 Oct 2010 at 2:45

**Comment by GoogleCodeExporter** [comment]
```
Sorry for ommitting, 1.4.0 Flash non-commercial build off your site.
```

Original comment by `Anson....@gmail.com` on 20 Oct 2010 at 2:50

**Comment by GoogleCodeExporter** [comment]
```
Not sure I understand your document; to mee it looks as if it contains 4 pages 
and the fourth page is smaller than the rest which is also what I see in 
FlexPaper when I load your document. FlexPaper is not showing page 4 as being 
as big as the rest
```

Original comment by `erik.eng...@devaldi.com` on 14 Nov 2010 at 10:32

**Comment by GoogleCodeExporter** [comment]
```
I guess you weren't able to duplicate the problem.  Let me try to explain it 
again:

The first 3 pages of the document are standard letter sized pages (8.5" x 11") 
in portrait orientation, and the 4th (last) page is a non-standard short size 
(8.5" x 0.8").  When the document loads, the combined height of the document 
should be 11" * 3 + 0.8" = 33.8", but instead I found the scroll bar scrolls 
past the end of the last page to the 44" mark, as if it assumed the last page 
is also letter sized.  

I have produced the problem with the attached file on both Windows/Firefox and 
Mac/Safari
```

Original comment by `Anson....@gmail.com` on 14 Nov 2010 at 9:08

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.4.2
```

Original comment by `erik.eng...@devaldi.com` on 11 Apr 2011 at 12:31
- Changed state: **Fixed**
